ArtGallery.Web.Api/Brokers/Navigations/NavigationBroker.cs

```diff
@@ -0,0 +1,19 @@
+﻿// -----------------------------------------------------------------------
+// Copyright (c) MumsWhoCode. All rights reserved.
+// -----------------------------------------------------------------------
+
+using Microsoft.AspNetCore.Components;
+
+namespace ArtGallery.Web.Api.Brokers.Navigations
+{
+    public class NavigationBroker : INavigationBroker
+    {
+        private readonly NavigationManager navigationManager;
+
+        public NavigationBroker(NavigationManager navigationManager) =>
+            this.navigationManager = navigationManager;
+
+        public void NavigateTo(String route) =>
```

Comment:
Use string data type than the String class

Reply:
resolved